### ISSUE TITLE
WorkflowKnowledgePublish metadata components are being grouped with WorkflowAlert

### ISSUE THREAD

**Issue by fehays** [scolladon/sfdx-git-delta#399]
### Discussed in https://github.com/scolladon/sfdx-git-delta/discussions/398

<div type='discussions-op-text'>

<sup>Originally posted by **fehays** December  7, 2022</sup>
When adding a Workflow for Knowledge__kav that contains a WorkflowKnowledgePublish, the package.xml that is generated groups this under the WorkflowAlert metadata types.
e.g.
```
    <types>
        <members>Knowledge__kav</members>
        <name>Workflow</name>
    </types>
    <types>
        <members>Knowledge__kav.PublishArticle</members>
        <name>WorkflowAlert</name>
    </types>
```
I end up with this error when trying to deploy the package:
```
Error Knowledge__kav.PublishArticle An object 'Knowledge__kav.PublishArticle' of type WorkflowAlert 
was named in package.xml, but was not found in zipped directory
```

I think the generated package.xml should put these items under a WorkflowKnowledgePublish element:
```
    <types>
        <members>Knowledge__kav</members>
        <name>Workflow</name>
    </types>
    <types>
        <members>Knowledge__kav.PublishArticle</members>
        <name>WorkflowKnowledgePublish</name>
    </types>
```

Is this a bug or am I doing something wrong?</div>

**Comment by scolladon** [comment]
Hi @fehays !

Thanks for raising this issue, thanks for contributing in making this project better.

I see what is the issue here
I am pushing a fix here #401.

Could you help us and do the User Acceptance please ?
You can do so by following [those steps](https://github.com/scolladon/sfdx-git-delta/blob/main/CONTRIBUTING.md#testing-the-plugin-from-a-pull-request)

**Comment by fehays** [comment]
Hi @scolladon ,
I've checked out the feature branch and installed the plugin.  Looks good!

I've confirmed the WorkFlowKnowledgePublish appears in the correct metadata type now:
```
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>Knowledge__kav</members>
        <name>Workflow</name>
    </types>
    <types>
        <members>Knowledge__kav.ArticleCustomAlert1</members>
        <members>Knowledge__kav.ArticleCustomAlert2</members>
        <name>WorkflowAlert</name>
    </types>
    <types>
        <members>Knowledge__kav.PublishArticle</members>
        <name>WorkflowKnowledgePublish</name>
    </types>
    <version>56.0</version>
</Package>
```

**Comment by scolladon** [comment]
Hi @fehays !

We just released the version `v5.10.0` containing the handling of those metadata.
You can install it using the `latest-rc` channel:

```sh
$ sfdx plugins:install sfdx-git-delta@latest-rc
````

Do not hesitate to create an issue if anything goes wrong using it.
Let us know as well if everything goes ok.
We will use your feedback to promote this version to the `latest` channel.
